src/components/beta-tag/BetaTag.tsx

```diff
@@ -10,7 +10,7 @@ export const BetaTag = ({style}: Props) => {
   const styles = useStyles();
   return (
     <View style={[styles.betaLabel, style]}>
```

Comment:
Could the `InfoChip` component have been used here? I know this is not your code originally, but when it's revisited I think it's a good idea to take the chance to improve the code. Generally, creating one-off UI elements (as is effectively done here) should not be done without really clear reasons.

Reply:
Yes, InfoChip can be used here as implemented in the new commit